Eugene-CG/popup/style.css

```diff
@@ -0,0 +1,109 @@
+* {
+	margin: 0;
+	padding: 0;
+	box-sizing: content-box;
+	list-style: none;
+    outline: none;
+}
+.wrapper {
+	width: 100%;
+	height: 300vh;
+	background-image: url(./img/gitlogoforme.jpeg);
+}
+.header {
+    margin: 0 auto;
+	max-width: 1198px;
+	height: 80px;
+	display: flex;
+	justify-content: space-between;
```

Comment:
Please, make your habbit to place css rules in logical order.
Please, fix your css rules style  according to one of this logic :
https://www.matuzo.at/blog/ordering-css-properties/
https://learn.javascript.ru/css-format#svoystva-silnee-vliyayuschie-na-dokument-idut-pervymi
https://www.antonball.dev/blog/2019-01-11-css-property-ordering/

Reply:
done